src/Config/Haskell.hs

```diff
@@ -78,22 +81,23 @@ readPragma o = case o of
         f _ _ = Nothing
 
 
-readComment :: Comment -> [Classify]
-readComment o@(Comment True _ x)
+readComment :: CommentEx -> [Classify]
+readComment c@CommentEx {ghcComment=L pos (AnnBlockComment s)}
```

Comment:
This will become `c@(L pos AnnBlockComment{})` in the new world.

Reply:
✔️